client/src/components/AuditAdmin/Progress/Progress.tsx

```diff
@@ -172,20 +178,18 @@ const Progress: React.FC<IProgressProps> = ({
             )
           case JurisdictionProgressStatus.UPLOADS_IN_PROGRESS:
             return <Status intent="warning">{filesUploadedText}</Status>
-          case JurisdictionProgressStatus.UPLOADS_NOT_STARTED:
-            return (
-              <Status>
-                {auditType === 'BALLOT_POLLING'
-                  ? 'No manifest uploaded'
-                  : filesUploadedText}
-              </Status>
-            )
+          case JurisdictionProgressStatus.UPLOADS_NOT_STARTED_LOGGED_IN:
+            return <Status intent="warning">Logged in</Status>
+          case JurisdictionProgressStatus.UPLOADS_NOT_STARTED_NO_LOGIN:
+            return <Status>Not logged in</Status>
           case JurisdictionProgressStatus.AUDIT_IN_PROGRESS:
             return <Status intent="warning">In progress</Status>
           case JurisdictionProgressStatus.AUDIT_COMPLETE:
             return <Status intent="success">Complete</Status>
-          case JurisdictionProgressStatus.AUDIT_NOT_STARTED:
-            return <Status>Not started</Status>
+          case JurisdictionProgressStatus.AUDIT_NOT_STARTED_LOGGED_IN:
+            return <Status intent="warning">Logged in</Status>
+          case JurisdictionProgressStatus.AUDIT_NOT_STARTED_NO_LOGIN:
+            return <Status>Not logged in</Status>
```

Comment:
Need to make sure the new added statuses sort correctly by updating `sortType` below